keylime/tenant.py

```diff
@@ -1162,26 +1174,31 @@ def do_add_allowlist(self, args):
         response = cv_client.post(
             f"/v{self.api_version}/allowlists/{self.ima_policy_name}", data=body, timeout=self.request_timeout
         )
-        Tenant._print_json_response(response)
+        Tenant._jsonify_response(response)
```

Comment:
This is kind of tricky to understand why we need it jsonify'd and it ends up printing it. Maybe don't print it in the function and do `print(Tenant._jsonify_response(response))` instead.

Reply:
My longer term plan is to get agreement from others to have every command output a JSON-formatted string that is easily consumable by tools such as `jq` (think of `do_cvstatus` as the model). However, I am trying to contain the number of changes on this PR without disrupting the existing functionality. Please consider that at some point `_jsonify_response` will json.dumps (maybe even with indent, to make it more human readable).

Reply:
Please note that I have an issue open with this purpose now https://github.com/keylime/keylime/issues/1159